python/packages/nisar/workflows/static.py

```diff
@@ -0,0 +1,378 @@
+#!/usr/bin/env python3
+
+from __future__ import annotations
+
+import argparse
+import os
+from collections.abc import Sequence
+from datetime import datetime, timezone
+from pathlib import Path
+
+import h5py
+import nisar
+from nisar.static.doppler import make_native_doppler_lut
+from nisar.static.ephemeris import get_cropped_orbit_and_attitude
+from nisar.static.geo_grid import get_output_geo_grid
+from nisar.static.geometry_layers import compute_geometry_layers
+from nisar.static.granule_id import form_granule_id
+from nisar.static.layover_shadow_mask import compute_geocoded_layover_shadow_mask
+from nisar.static.logging import get_logger, log_elapsed_time
+from nisar.static.product import (
+    build_hdf5_dataset_creation_kwds_dict,
+    populate_grids_group,
+    populate_identification_group,
+    populate_metadata_group,
+)
+from nisar.static.rtc_anf_layers import compute_rtc_anf_layers
+from nisar.static.runconfig import get_runconfig_params
+from nisar.static.util import get_raster_dataset_metadata_item, scratch_directory
+from nisar.static.water_mask import binarize_and_reproject_water_mask
+
+import isce3
+from isce3.geometry import make_geo_grid_bounding_polygon
+
+
+def run_static_layers_workflow(config_file: os.PathLike | str) -> None:
+    """
+    Run the NISAR Static Layers workflow with the specified run configuration file.
+
+    Parameters
+    ----------
+    config_file : path-like
+        The file path of the runconfig YAML file.
```

Comment:
Pendatic, but can we make this docstring specific to a runconfig yaml file for STATIC layers? (i.e. not an RSLC runconfig yaml?)

I don't like this phrasing, but to give an idea of what I'm asking:

```suggestion
        The file path to a STATIC layers runconfig YAML file.
```